app/uk/gov/hmrc/excisemovementcontrolsystemapi/models/auditing/MessageSubmittedDetails.scala

```diff
@@ -16,8 +16,11 @@
 
 package uk.gov.hmrc.excisemovementcontrolsystemapi.models.auditing
 
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.functional.syntax.unlift
+import play.api.libs.json.{JsObject, JsValue, Json, OWrites, Reads, Writes, __}
 import cats.data.NonEmptySeq
-import play.api.libs.json.{JsObject, Json}
+import play.api.libs.functional.syntax.unlift
```

Comment:
Is this import needed?

Reply:
Yes, we need it for the writer to work. We unlift the unapply.